src/types.rs

```diff
@@ -157,15 +159,23 @@ pub struct ObfuscatedTunnelConfig {
     // TODO: remove after roll out in all environments
     #[serde(default = "default_gateway_ip_v4")]
     pub gateway_ip_v4: net::Ipv4Addr,
+    // TODO: https://linear.app/soveng/issue/OBS-1267
```

Comment:
Nit:

```suggestion
    // TODO: Remove. https://linear.app/soveng/issue/OBS-1267
```

Or just `// LEGACY: https://...`?

Reply:
I opted for `TODO: Remove.`. My editor doesn't highlight `LEGACY:`